client/packages/cli_new/src/commands/set_operational.ts

```diff
@@ -0,0 +1,61 @@
+import ora from "ora"
+import { createType } from "@t3rn/types"
+import { Gateway } from "@/schemas/setup.ts"
+import { createCircuitContext } from "@/utils/circuit.ts"
+import { getConfig } from "@/utils/config.ts"
+import { colorLogMsg, log } from "@/utils/log.ts"
+
+export const spinner = ora()
+
+export const handleSetOperational = async (id: string, enabled: boolean) => {
+  const config = getConfig()
+
+  if (!config) {
+    process.exit(1)
+  }
+
+  const gateway = config.gateways.find((elem) => elem.id === id)
+
+  if (!gateway) {
+    log("ERROR", `Gateway with id ${id} not found in config`)
+    process.exit(1)
+  }
+
+  spinner.start("Setting gateway operational...")
+
+  try {
+    const transactionArgs = setGatewayOperational(gateway, enabled)
+    const { circuit, sdk } = await createCircuitContext()
+    const transaction = circuit.tx.portal.setOperational(
+      transactionArgs.gatewayId,
+      transactionArgs.operational
+    )
+
+    await sdk.circuit.tx.signAndSendSafe(sdk.circuit.tx.createSudo(transaction))
+    spinner.stopAndPersist({
+      symbol: "🚗",
+      text: colorLogMsg(
+        "SUCCESS",
+        `Gateway ${id} operational status set to ${enabled}`
+      ),
+    })
+  } catch (error) {
+    spinner.fail(colorLogMsg("ERROR", error))
+    process.exit(1)
+  }
+}
+
+export const setGatewayOperational = (gateway: Gateway, enabled: boolean) => {
+  switch (gateway.registrationData.verificationVendor) {
```

Comment:
I dont think we need to match between different verificationVendors here, portal should handle this for every gateway

Reply:
Fixed; please refer to commit [1c055e0](https://github.com/t3rn/t3rn/pull/881/commits/1c055e047c4f33439b6c8b2535faaeeef707aef0).